packages/jest-console/src/__tests__/CustomConsole.test.ts

```diff
@@ -186,4 +186,13 @@ describe('CustomConsole', () => {
       expect(_stdout).toMatch('ms');
     });
   });
+
+  describe('dir', () => {
+    test('should print the deepest value', () => {
+      const deepObject = {1: {2: {3: {4: {5: {6: 'value'}}}}}};
+      _console.dir(deepObject, {depth: 6});
+
+      expect(_stdout).toMatch('value');
```

Comment:
maybe this should be a snapshot so we can see the entire structure? same for the test below

Reply:
`util.inspect` implementation may differ from version to version as described in node.js documentation, but if that's ok for you, I'll introduce `matchInlineSnapshot`.

Reply:
Jest's output in this case shouldn't vary though, should it?

Reply:
Could you, please, rephrase the sentence? Not sure I properly understand you.

`util.format` may also differ from version to version, thus Jest's `console.log` output may vary from version to version. Same with `console.dir`.

Will Node.js core developers decide to change `util.inspect` for deep objects? Who knows, maybe they will add/remove extra spaces between colons. But `value` will always be in the output.

I'm not a core developer, so just say what to do, I'll do it.